remotelog/loki.go

```diff
@@ -124,6 +125,9 @@ func (l *Loki) start(hCh chan *decoder.HEP) {
 				l.entry.labels["protocol"] = model.LabelValue(protocol)
 			}
 
+			hostname, _ := os.Hostname()
```

Comment:
1) It's best practice in Go to always check the error.
2) The os.Hostname() call ist expensive and just needs to be done once so move it out of the for loop.

Reply:
Thanks @negbie,  updated